src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/AppxPackageHelper.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
 using System.Runtime.InteropServices.ComTypes;
-using static Microsoft.Plugin.Program.Programs.UWP;
+using HRESULT = Wox.Plugin.Common.Win32.HRESULT;
```

Comment:
i don't like an enum and namespace sharing.  This can cause easy conflicting.  I don't see why HRESULT is part of the namespace here as well if it is just the enum.

This feels odd how it is written.

For c#, pretty sure this would still be written Hresult.  https://docs.microsoft.com/en-us/dotnet/api/system.exception.hresult?view=net-6.0

Reply:
what do you mean an enum and namespace sharing?
`HRESULT` is in `Wox.Plugin.Common.Win32`.

>For c#, pretty sure this would still be written Hresult. https://docs.microsoft.com/en-us/dotnet/api/system.exception.hresult?view=net-6.0

Do you mean is should rename it to `HResult`?

Reply:
There are two things here.  First is changing from HResult to HRESULT

Second and larger part is having a namespace forced to be the same as the enum.  My mind is slightly blown this compiles.

Reply:
>There are two things here. First is changing from HResult to HRESULT

`HResult` was the enum defined by this plugin. I named the common one `HRESULT` to match the win32 name. (like the other win32 enums and structs defined).

>Second and larger part is having a namespace forced to be the same as the enum. My mind is slightly blown this compiles.

[This is a using alias, not a namespace.](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#using-alias) `Wox.Plugin.Common.Win32` is the namespace.